src/main/resources/avro/action.avsc

```diff
@@ -53,6 +55,14 @@
       },
       "default": []
     },
+    {
+      "name": "params",
+      "type": {
+          "type": "string",
+          "logicalType": "json-object"
```

Comment:
@pilhuhn Now params is a serialized JSON object. 
Created some classes to do the automagic transformation from string to HashMap (and viceversa). If the implementor app doesn't have it, it can write it directly as a string (manually doing the json encode/decode) or write the same bindings.

WDYT?